### DISCUSSION TITLE
Общее обсуждение

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/gothic2_steam_fix#16] [General]
а что это за плюсики зеленые у меня появились?
![image](https://user-images.githubusercontent.com/49514764/94133544-766b8300-fe69-11ea-9ce4-5d7ea1d21591.png)

**Comment by dosinabox** [comment]
Это отметка новых предметов. Выключается в настройках игры (Опции - Плагины).

**Comment by Kor-Angar** [comment]
что то не нашёл. в самой игре?

**Comment by dosinabox** [comment]
да, в игре, из главного меню

**Comment by Kor-Angar** [comment]
![image](https://user-images.githubusercontent.com/49514764/94144713-2ba63700-fe7a-11ea-9594-1b919dc8d687.png)

**Comment by Kor-Angar** [comment]
наверное я опять неправильно устанавливал через СтимФикс :-)

**Comment by Kor-Angar** [comment]
переустановил. оказывается надо было галку про русскую версию ставить. соответственно если без русской версии устанавливать то в меню про плагины нет.

**Comment by Kor-Angar** [comment]
======================================= UNHANDLED EXCEPTION OCCURED ======================================================
======================================= CRASH INFOS: =====================================================================
Gothic II - 2.6 (fix), Parser Version: 50
User:  USER,  CPUType: 586,  Mem: 0 MB total, 0 MB free
Startup Options:-game:_g2a_nr_scriptpatch_v25.ini
=============================================== CALLSTACK : ==============================================================
0023:004DBAC0 (0x0135FB40 0x0135FC58 0x0082E6F0 0x0082E600) GothicMod.exe, zCMenu::Run()+288 byte(s), P:\dev\g2addon\release\ZenGin\_carsten\zMenu.cpp, line 451i
0023:6C560269 (0x004C7270 0x00402210 0x00401F40 0x00401F50) MSVCR100.dll, malloc()+54 byte(s)

**Comment by dosinabox** [comment]
https://remontka.pro/msvcr100-dll-otsutstvuet/

**Comment by Kor-Angar** [comment]
![image](https://user-images.githubusercontent.com/49514764/94147095-84c39a00-fe7d-11ea-9205-2e9c6af98062.png)
а тут какой лишний?

**Comment by dosinabox** [comment]
Неофициальное обновление распространяется как отдельным модом, так и в составе стим-фикса. Для стим-фикса я делаю некоторые изменения:
_autoscale.mod для стима лишний
_speech_fix.mod уже включен в озвучку
_widescreen.mod уже включен в русифицированные текстуры
_wasteland.mod в стим-фикс не включен, можешь докинуть его отдельно и он подхватится игрой

Пожалуйста, удали игру через стим, удали её папку, установи чистую игру заново, поставь сверху сентябрьский стим-фикс, дополни классикой, шрифтами и dx11. И больше ничего не трогай)

**Comment by Kor-Angar** [comment]
а файлы игры остаются оригинальными или тоже меняются? 

![image](https://user-images.githubusercontent.com/49514764/94231413-a28c1000-ff0c-11ea-815d-c1a755f85f12.png)

**Comment by Kor-Angar** [comment]
> Пожалуйста, удали игру через стим, удали её папку, установи чистую игру заново, поставь сверху сентябрьский стим-фикс, дополни классикой, шрифтами и dx11. И больше ничего не трогай)

так тогда всё будет хорошо и прекрасно. себе то сделать легко, другим сложно ;-)

**Comment by dosinabox** [comment]
> а файлы игры остаются оригинальными или тоже меняются?

Скрипты стим-версии заменяются на оригинальные скрипты русской версии от Акеллы без каких-либо изменений. Если не выбирать опцию русификации текста, то остаются из стим-версии. Также оригинал не использует контент неофициального обновления и плагины Union. Из измененных файлов для оригинала только несколько русифицированных текстур, широкоформатные загрузочные экраны и фон главного меню из Textures_Russian.vdf.

Что за `G2 NotR Unofficial v`? Это не мое)

**Comment by Kor-Angar** [comment]
> Скрипты стим-версии заменяются на оригинальные скрипты русской версии от Акеллы без каких-либо изменений. Если не выбирать опцию русификации текста, то остаются из стим-версии.

и  это прекрасно :-)

> Что за G2 NotR Unofficial v? Это не мое)

это я подписал для себя. 

![image](https://user-images.githubusercontent.com/49514764/94234402-47f5b280-ff12-11ea-8b52-b57fba625dd1.png)
текст можно будет поменять,  ссылка там битая

**Comment by Kor-Angar** [comment]
кажется в  интернациональной версии в файле GothicGame_original.ini указан не тот файл с описанием. 
сейчас идёт Description=!<symlink>**GothicGame**.rtf , а должно быть Description=!<symlink>**GothicGame_original**.rtf

проверь

**Comment by dosinabox** [comment]
В установщик пакуется GothicGame_original.rtf. При установке он переименовывается в GothicGame.rtf.

**Comment by Kor-Angar** [comment]
подскажи пожалуйста а что за файл такой GothicGame.mod который указан в ini файле?

**Comment by dosinabox** [comment]
Вроде как параметр VDF в ini мода/игры не может быть пустым, поэтому разработчики дописывали туда вот такую несуществующую заглушку.

**Comment by Kor-Angar** [comment]
Но у меня это файл есть в папке со стимовской версии. А у тебя в стимовской версии он есть?

**Comment by dosinabox** [comment]
А что внутри, сколько весит?

**Comment by dosinabox** [comment]
перепроверь ссылку, не загрузилось

**Comment by Kor-Angar** [comment]
[GothicGame.zip](https://github.com/dosinabox/g2_steam_fix/files/5347123/GothicGame.zip)

**Comment by Kor-Angar** [comment]
может появляется при установке G2NotR MDK ?

**Comment by dosinabox** [comment]
> может появляется при установке G2NotR MDK ?

Именно так. А внутри немецкие скрипты. И поэтому неумелые пользователи потом жалуются, что у них игра стала на немецком.
При установке одного только Player Kit этот файл не устанавливается, а запись в ini остается.

**Comment by Kor-Angar** [comment]
> И поэтому неумелые пользователи потом жалуются, что у них игра стала на немецком.

неумелые пользователя доверяют умелым умелым **М**астерам. а умелые Мастера добавляют такую хрень в свои ini файлы в рамках инсталлеров к СтимФикс,  подключая немецкие скрипты и полякам и русским и тем у кого яндекс.диск не работает.. но не всем конечно, а только неумелым пользователям :-D

**Comment by dosinabox** [comment]
Откуда эти немецкие скрипты у юзера берутся, и почему их просто не удалить за ненадобностью (раз уж взялись) - отдельный вопрос. Установить можно много чего, а потом удивляться.

**Comment by Kor-Angar** [comment]
что бы их удалять нужно знать что они есть. они могут взяться в рамках какой то сборке или после установки какого то мода или просто кто кому то скинул архив и предложил скопировать. вообщем есть пользователи у них есть такая вот история. можно им помочь, а можно их ситуацию усугубить. используя право **М**астера. очевидно же что пользователь дебил. и он скорее всего один. а если даже их много он же всё равно кретин ;-) а если среди этих кретинов, сть нормальный пользователь то он же должен включить мозг. ну и так далее :-) я думаю это всё подробно по телевизору нынче рассказывают.

**Comment by dosinabox** [comment]
Какой версией DX11 ты пользуешься?

**Comment by Kor-Angar** [comment]
У меня последняя стоит. 12:15, 20 октября 2020 г., dosinabox <notifications@github.com>:
Какой версией DX11 ты пользуешься?

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Отправлено из мобильного приложения Яндекс.Почты

**Comment by dosinabox** [comment]
Этот? https://github.com/Kirides/GD3D11

**Comment by Kor-Angar** [comment]
Pre-release  v17.7-dev12
![image](https://user-images.githubusercontent.com/49514764/96577706-af6a0c80-12dc-11eb-868f-e3d7c0827d2c.png)

у меня он работает корректно. ошибок нет. только призраки не работают это из того что в глаза бросаются.

**Comment by dosinabox** [comment]
> у меня он работает корректно. ошибок нет.

А у меня нет - требует оригинальный Gothic2.exe. И люди жалуются.
Подскажешь, в чем может быть дело?

![image](https://user-images.githubusercontent.com/39790055/96578360-c0ffe400-12dd-11eb-94e2-2c16cef47ded.png)

**Reply by Kor-Angar** [reply]
> А чтобы этого предупреждения не было, нужно чтобы в папке system был файл Gothic.exe репорт-версии 2.6 без модификации для расширения памяти. Потом сделаю.


https://github.com/dosinabox/g2_steam_fix/issues/12

**Comment by Kor-Angar** [comment]


**Reply by dosinabox** [reply]
Убери все файлы _plugins.mod из Data/ModVDF, достань все плагины из _v26_plugins.mod (.dll и .patch) и копируй их в System/Autorun по одному и проверяй, на каком начнутся лаги.

**Reply by dosinabox** [reply]
zStepHeight.dll проверь, но не используй, его не будет в релизе 27 версии. Надо будет доделывать.

**Comment by bezgin750** [comment]
Привет, а можно как то установить fix на линукс версию готики2?

**Reply by dosinabox** [reply]
Под Wine должно установиться и даже заработать, но лично проверить не могу. Подробнее посмотри здесь: https://worldofplayers.ru/threads/40376/  (раздел "Запуск под Wine (или другими (не)эмуляторами)")

**Reply by bezgin750** [reply]
смог поставить через PortProton, но выскакивает ошибка, если игнорировать игра работает но нету катцен
![i0AvHJ4dUCE](https://user-images.githubusercontent.com/91847262/138739146-7294d5d5-58a5-4459-82a6-3e209e9076ba.jpg)

**Reply by dosinabox** [reply]
Игнорируй, ничего страшного. Или удали файл zGamePad.vdf, этот плагин у тебя всё равно не работает.
А катсцен нету не поэтому. Попробуй BorderlessWindow=1 в system/SystemPack.ini. Если не поможет, то ещё FixBink=0.

**Reply by bezgin750** [reply]
все сломалось теперь просто черный экран) возвращаю параметры обратно со ровно черный экран

**Reply by dosinabox** [reply]
Удали файл system/autorun/AutoScreenRes.dll

**Reply by bezgin750** [reply]
не помогло, восстановил файлы, починилось но нету катцен и выбора версии, русский язык остался

**Reply by dosinabox** [reply]
Возможно, восстановилось не всё. Обновленный Gothic2.exe должен запускать стартер модов с выбором версии. Оригинальный сразу запускает только основную игру.

**Comment by Kor-Angar** [comment]
Привет. а ты в чём инсталлер делаешь?

**Reply by dosinabox** [reply]
NSIS

**Reply by dosinabox** [reply]
3.0, если точнее

**Reply by Kor-Angar** [reply]
а я уже в винраре сделал. у меня ничего там сложно не оказалось и так пойдёт :-)

**Reply by Kor-Angar** [reply]
только не допендрил как путь к стиму найти

**Reply by Kor-Angar** [reply]
а почему у тебя инсталлер такой путь предлагает? 
![image](https://user-images.githubusercontent.com/49514764/141003718-bf12b01d-feca-4552-94c0-5bd6344bcf64.png)

**Reply by Kor-Angar** [reply]
Мюллер предложил Штирлицу работать на него. «Но я уже работаю на Шелленберга. — Как быть с трудовой книжкой?» — «А если завести вторую?» — предложил Мюллер. «Знаю я ваши штучки, — не согласился Штирлиц. — Вы потом пришлете гестаповцев проверить, сколько у меня трудовых книжек». — «С вами страшно разговаривать, Штирлиц, — признался Мюллер. — Вы просто читаете мои мысли». 

![image](https://user-images.githubusercontent.com/49514764/141004168-bb13d86a-1970-4589-bbd0-017ff6c31eca.png)

**Reply by dosinabox** [reply]
Лучше Akella Games для русской версии и JoWooD для английской. Но вообще не угадаешь:

```
Gothic II Gold Edition (2005)[PCDVD-RUS-Akella]
C:\Program Files\Akella Games\Gothic II
Akella Games\Gothic II


Gothic II Gold Edition (2007)[PCDVD-RUS-Akella]
C:\Program Files\Akella Games\Gothic II
Akella Games\Gothic II


Gothic II Gold Edition (2005)[PC5CD-RUS-Akella]
C:\Program Files\Akella Games\Gothic II
Akella Games\Gothic II


Gothic II (2004)[PCDVD-RUS-Akella]
C:\Program Files\Akella Games\Gothic II
Akella Games\Gothic II


Gothic II (2004)[PC3CD-RUS-Akella]
C:\Program Files\Akella Games\Gothic II
Akella Games\Gothic II


Gothic 2 Gold Edition (2005)[ENG-GOG]
C:\Program Files\GOG.com\Gothic 2 Gold
GOG.com\Gothic 2 Gold


Gothic 2 Gold Edition (2015)[MULTi3-GOG]
C:\GOG Games\Gothic 2 Gold
GOG.com\Gothic 2 Gold


Gothic 2 Gold Edition (2017)[MULTi6-GOG]
C:\GOG Games\Gothic 2 Gold
GOG Games\Gothic 2 Gold


Gothic II Gold - (2005)[PCDVD-ENG-JoWooD]
C:\Program Files\JoWooD\Gothic II
JoWooD\Gothic II


Gothic II Gold - (2005)[PCDVD-DEU-JoWooD]
C:\Program Files\JoWooD\Gothic II
JoWooD\Gothic II


Gothic II Gold - (2004)[PC4CD-ITA-TNT VILLAGE] (Espansione La Notte del Corvo)
C:\Program Files\JoWooD\Gothic II
JoWooD\Gothic II


Gothic II Gold Edition (2017)[PCDVD-MULTi6-PROPHET]
C:\Program Files\THQ Nordic\Gothic II - Gold Edition
THQ Nordic\Gothic II - Gold Edition


Gothic Complete Collection English [PC3DVD5-DEFA]
C:\Program Files\JoWooD Productions Software AG\Gothic II Gold
JoWooD\Gothic II Gold
```

**Reply by Kor-Angar** [reply]
так а на англ. инсталлере у тебя путь идёт к Акелле.

**Reply by dosinabox** [reply]
да, мой косяк

**Reply by Kor-Angar** [reply]
только сейчас заметил в описание что ему плееркит хочется

**Reply by dosinabox** [reply]
а что не так?

**Reply by Kor-Angar** [reply]
а у меня он не хотел вставать на чистую версию, а я понять не мог чего он не хочет становится :=)

**Reply by dosinabox** [reply]
ты про стим-фикс или обновление отдельным модом?

**Reply by Kor-Angar** [reply]
мод версию. в этом окошке обычно пишут что надо указать путь к папке и глаз проскакивает плеер кит.

**Comment by Kor-Angar** [comment]
```
[MOD]
BASE=g123
```

**Reply by Kor-Angar** [reply]
![image](https://github.com/dosinabox/g2_steam_fix/assets/49514764/f100071d-8129-42be-accd-0c8bd2ea7013)
